server/models/ClubProfileModel.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const clubProfileSchema = new Schema({
+    name: { type: String, required: true },
+    description: { type: String, required: true },
+    lastUpdated: { type: Date, default: Date.now() },
+    imageUrl: { type: String, default: '' },
+    memberRange: { type: String, required: true },
+    acceptingMembers: { type: Boolean, required: true },
```

Comment:
in the registration form, there's a question "when do you take new members", to which the club answers either "fall" and/or "spring", or "currently not accepting new members". Let's try to store all of that information instead of simply a boolean value, since we can always parse the data in the backend and send a final definitive "yes" or "no" to the frontend.